src/cgame/default/cg_temp_entity.c

```diff
@@ -298,8 +301,9 @@ void Cg_GibEffect(const vec3_t org, int32_t count) {
 
 	cgi.AddStain(&(const r_stain_t) {
 		.origin = { org[0], org[1], org[2] },
-		 .color = { 0.6 + 0.1 * Randomc(), 0.0, 0.0, 0.9 + Randomf() * 0.1 },
-		  .radius = count * 4.0
+		.radius = count * 4.0,
+		.image = cg_particles_normal->image,
+		.color = { 0.6 + 0.1 * Randomc(), 0.0, 0.0, 0.9 + Randomf() * 0.1 },
```

Comment:
Yea, `astyle` does this weird incremental indent thing on these initializers. Very strange.